dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditFieldAction.java

```diff
@@ -312,12 +320,16 @@ private boolean _saveField(ActionForm form, ActionRequest req, ActionResponse re
                     }
 
                   }catch (Exception e) {
-                      String message = "message.structure.invaliddata";
-                    SessionMessages.add(req, "error", message);
+                    SessionMessages.add(req, ERROR, INVALIDATE_DATA_MESSAGE);
                     return false;
                  }
             }
 
+            // Validate values entered for decimal/number/boolean type radio button field
+            if (field.getFieldType().equals(FieldType.RADIO.toString()) && !isValidRadioField(fieldForm,req)){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "FieldType"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "RADIO"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)